sdk/python/foundation-models/system/inference/README.md

```diff
@@ -0,0 +1,22 @@
+# Foundation Model Inferencing
+The __foundation-model-inference__ container is a curated solution for inferencing foundation models on Azure ML. It incorporates the best inferencing frameworks to ensure optimal request throughput and latency. The container is user-friendly and integrates seamlessly with Azure ML. It utilizes:
+
+## vLLM
+vLLM is a high-performance inferencing server that offers several features, making it a top choice for inferencing systems. vLLM provides:
+- Top-tier serving throughput
+- Efficient handling of attention key and value memory with PagedAttention
+- Continuous batching of incoming requests
+- Optimized CUDA kernels
+- High-throughput serving with various decoding algorithms, including parallel sampling, beam search, and more
+- Support for tensor parallelism for distributed inference
+
+## DeepSpeed FastGen
+DeepSpeed FastGen, a recent release from DeepSpeed, offers up to 2.3x faster throughput than vLLM, which already outperforms similar frameworks like Huggingface TGI. DeepSpeed-FastGen combines DeepSpeed-MII and DeepSpeed-Inference to deliver a fast and user-friendly serving system.
```

Comment:
do we want to emphasize similar things as in vllm section? high serving throughput, batching, optimized CUDA kernels, tensor parallelism and on top of it Dynamic Split-Fuse?